server/server.js

```diff
@@ -32,38 +32,40 @@ app.use(
 
 const port = process.env.PORT || 4000;
 const configPath = process.env.MOBILE_SERVICES_CONFIG_FILE || '/etc/mdc/servicesConfig.json';
+const dataSyncService = {
+  type: DataSyncService.type,
+  mobile: true
+};
 const DEFAULT_SERVICES = {
-  services: [
+  version: 'dev',
+  components: [
     {
       type: IdentityManagementService.type,
-      url: `https://${process.env.IDM_URL || process.env.OPENSHIFT_HOST}`
+      version: 'latest',
+      host: `https://${process.env.IDM_URL || process.env.OPENSHIFT_HOST}`,
+      mobile: true
     },
     {
       type: PushService.type,
-      url: `https://${process.env.UPS_URL || process.env.OPENSHIFT_HOST}`
+      host: `https://${process.env.UPS_URL || process.env.OPENSHIFT_HOST}`,
+      version: 'latest',
+      mobile: true
     },
     // {
     //   type: MetricsService.type,
     //   url: `https://${process.env.METRICS_URL || process.env.OPENSHIFT_HOST}`
     // }
-    {
-      type: DataSyncService.type
-    },
     {
       type: MobileSecurityService.type,
-      url: `https://${process.env.MSS_URL || process.env.OPENSHIFT_HOST}`
+      host: `https://${process.env.MSS_URL || process.env.OPENSHIFT_HOST}`,
```

Comment:
If you set `OPENSHIFT_HOST` to include the protocol (e.g `https://example-url.com`) then this host will be set to `https://https://example-url.com` if `MSS_URL` is not set.

The same would occur with the other services that have a host key.

![Screenshot from 2019-07-17 15-29-41 (copy)](https://user-images.githubusercontent.com/11743717/61384058-bb7c4880-a8a7-11e9-9e76-0c81c0385a66.png)